tests/Service/PromotionsService/FindActiveSuperPromotionsByProgrammeTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Service\PromotionsService;
+
+use DateTimeImmutable;
+
+class FindActiveSuperPromotionsByProgrammeTest extends AbstractPromotionServiceTest
```

Comment:
Test name should match the name of the function you're testing `FindActiveSuperPromotionsByAncestryTest`